core/src/main/java/org/infinispan/stats/impl/AbstractClusterStats.java

```diff
@@ -163,7 +163,17 @@ int addIntAttributes(List<Map<String, Number>> responseList, String attribute) {
    }
 
    void putLongAttributesAverage(List<Map<String, Number>> responseList, String attribute) {
-      long average = addLongAttributes(responseList, attribute) / responseList.size();
+      long nonZeroValues = 0;
+      long total = 0;
+      for (Map<String, Number> m : responseList) {
+         Number value = m.get(attribute);
+         long longValue = value.longValue();
+         if (longValue > 0) {
```

Comment:
@vblagoje No, that is resolved by the changes in https://github.com/infinispan/infinispan/pull/5110/commits/57497d61ddd7efc3084b654c59021c3e88456c85. 

Here we need the recorded time to be greater than zero, otherwise no latency has been recorded, so that when calculating the average we only divide by the number of nodes that have actually recorded a latency.